### DISCUSSION TITLE
Uncaught TypeError: this.$wire is undefined - getActiveComponentModalAttribute

### DISCUSSION THREAD

**Discussion by hundra** [wire-elements/modal#395] [Q&A]
I have a fresh installation with Laravel 10.10,  Breeze 1.25, Livewire 3.1 and wire-elements/modal 2.0.8.

After I have added @livewire('wire-elements-modal') to my app.blade.php I get a console error saying:

"Uncaught TypeError: this.$wire is undefined" related to "getActiveComponentModalAttribute".

The model works but I cannot get ride of the error.

**Comment by x-mago** [comment]
I have the same Problem. One suggestion to solve this, I found several times, is to put the livewireScripts before Alpine. But this has no effect.

**Comment by weobleyc** [comment]
I have the same issue. Don't know where to start.
